benchmarks/templates/benchmarks/faq.html

```diff
@@ -0,0 +1,101 @@
+      <span class="benefits_heading"><h3 class="benefits_heading is-size-3-mobile">I want to use this; how do I get started? </h3></span>
+      <span class="icon is-small"><i class="fas fa-caret-down icon-gradient"></i></span>
+    </button>
+    <div class="content is-hidden">
+        <p class="benefits_info is-size-5-mobile question_text has-text-weight-bold no_top_header">
+            6) What am I able to submit?
+        </p>
+        <p class="benefits_info is-size-5-mobile shorter no_top">
+            You are able to submit a compressed directory containing an __init__.py, a setup.py file, a model file, and a test.py file. The __init__.py file adds your model to the Brain-Score registry. The tests.py file can remain blank for submission. The other two files are optional, but the setup.py file outlines the requirements that your model has. The model.py file defines the model itself.
+        </p>
+        <p class="benefits_info is-size-5-mobile question_text has-text-weight-bold no_top_header">
+            7) Am I able to keep my submissions private?
+        </p>
+        <p class="benefits_info is-size-5-mobile shorter no_top">
+            With the Brain-Score 2.0 release, we do now support private submissions!
+        </p>
+        <p class="benefits_info is-size-5-mobile question_text has-text-weight-bold no_top_header">
+            8) My submission is not being accepted. Why?
+        </p>
+        <p class="benefits_info is-size-5-mobile shorter no_top">
+            There are a variety of reasons that your submission is not being accepted. The best starting point is ensuring that the checks on your submission are passing. Then reach out via Slack explaining your problem and someone in our developer community will assist you in passing these checks.
+        </p>
+        <p class="benefits_info is-size-5-mobile question_text has-text-weight-bold no_top_header">
+            9) My submission passed. What now?
+        </p>
+        <p class="benefits_info is-size-5-mobile shorter no_top">
+            Once the accepted submission has been processed, you should be seeing a Brain-Score of your model on the leaderboard now! If you have any further questions, please don't hesitate to reach out to our developer community in Slack.
+        </p>
+    </div>
+</div>
+<div class="faq_box leaderboard-table-component">
+    <button class="toggle-button is-size-3-mobile" aria-expanded="false">
+      <span class="benefits_heading"><h3 class="benefits_heading is-size-3-mobile">Common problems with scoring and submissions</h3></span>
+      <span class="icon is-small"><i class="fas fa-caret-down icon-gradient"></i></span>
+    </button>
+    <div class="content is-hidden">
+        <p class="benefits_info is-size-5-mobile question_text has-text-weight-bold no_top_header">
+            10) My model's GitHub PR is ready to merge but hasn't been. Who should I reach out to?
```

Comment:
Is that a bad assumption to make? In theory I'm imagining a new user will be coming here after they get stuck on a Deep Dive tutorial step, which implies that they have at least started this process.

If the assumption is completely a brand-new, never-tried-a-tutorial BrainScore user, then this doesn't fit here.

Reply:
There's no mention in the Deep Dive tutorial of what happens to your submission after it's uploaded. A user who submits via the website would typically just expect results to show up in their profile / on the leaderboard and have no knowledge that GitHub is involved in any way. It would be good to add an FAQ question along the lines of "what happens to my submission after I upload it?" that explains the submission pipeline, as I don't believe it's currently documented anywhere outside of the developer documentation (which would be helpful to link to in the answer to that question).

Reply:
Removed this section.